spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,24 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_name, query)
```

Comment:
No thats okay. Can you change the docs of `from_sql` to reflect the fact that passing a string also works now?